graphql_compiler/macros/macro_edge/validation.py

```diff
@@ -36,6 +36,8 @@ def _validate_macro_ast_with_macro_directives(schema, ast, macro_directives):
             u'Unexpectedly found variable definitions at the top level of the GraphQL input. '
             u'This is not supported. Variable definitions: {}'.format(ast.variable_definitions))
 
+    # TODO(vlad): this should be replaced with get_schema_for_macro_definitions
+    #             without creating circular dependencies.
```

Comment:
nit: "Replace this get_schema_for_macro_defintions without creating circular dependencies"